djredcap/management/subcommands/inspect.py

```diff
@@ -50,6 +51,9 @@ class Command(BaseCommand):
     requires_model_validation = False
     db_module = 'django.db'
     args = 'filename'
+    #add a command line option -m allowing the user to set a filename for models
```

Comment:
Fix grammar in comments (i.e. capitalize the first word). Add space after pound sign `# Add a ...`